xml/ha_maintenance.xml

```diff
@@ -0,0 +1,548 @@
+      </para>
+     </note>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="vle.ha.maint.mode.node">
+    <!--<term>Putting a Node Into Maintenance Mode</term>-->
+    <term><xref linkend="sec.ha.maint.mode.node" xrefstyle="select:title"/></term>
+    <listitem>
+     <para>
+      Use this option if you want to do maintenance for plenty of (or all)
+      resources hosted on a node. It can also be used for updating software
+      packages on a node that do <emphasis>not</emphasis> affect the cluster
+      stack. See also <xref linkend="sec.ha.migration.update"/>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="vle.ha.maint.node.standby">
+    <!--<term>Putting a Node into Standby Mode</term>-->
+    <term><xref linkend="sec.ha.maint.node.standby" xrefstyle="select:title"/></term>
+    <listitem>
+     <para>
+      A node that is in standby mode can no longer run resources. Any resources
+      running on the node will be moved away or stopped (in case no other node
+      is eligible to run the resource). Also, all monitor operations will be
+      stopped on the node (except for those with
+      <literal>role="Stopped"</literal>).
+     </para>
+     <para>
+      You can use this option if you need to stop a node in a cluster while
+      continuing to provide the services running on another node.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="vle.ha.maint.rsc.unmanaged">
+    <!--<term>Putting a Resource Into Unmanaged Mode</term>-->
+    <term><xref linkend="sec.ha.maint.rsc.unmanaged" xrefstyle="select:title"/></term>
+     <listitem>
+     <para>
+      The <option>is-managed</option> meta attribute allows you to temporarily
+      <quote>release</quote> a resource from being managed by the cluster
+      stack. This means you can manually touch the service that is managed by
+      this resource (for example, to adjust any components). However, the
+      cluster will continue to <emphasis>monitor</emphasis> the resource and to
+      report any failures.
+     </para>
+     <para>
+      If you want the cluster to also cease <emphasis>monitoring</emphasis> the
+      resource, use the per-resource maintenance mode instead (see
+      <xref 
+      linkend="vle.ha.maint.mode.rsc"/>).
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="vle.ha.maint.mode.rsc">
+    <!--<term>Putting a Resource Into Maintenance Mode</term>-->
+    <term><xref linkend="sec.ha.maint.mode.rsc" xrefstyle="select:title"/></term>
+    <listitem>
+     <para>
+      When this mode is enabled for a resource, no monitor operations will be
+      triggered for the resource.
+     </para>
+     <para>
+      Use this option if you need to manually touch the service that is managed
+      by this resource and do not want the cluster to run any monitoring
+      operations for the resource during that time.
+     </para>
```

Comment:
Maybe it doesn't really matter. But since "maintenance mode" is the preferred way, how about we introduce this before "unmanaged mode"? The description currently in "unmanaged mode" apply to "maintenance mode" except the "monitor" part.